### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by davidhao3300**
This moves all the page groups to a YAML file, and updates the README.

### CODE REVIEW

File: cal/fb.py [hunk 1]
```diff
@@ -1,108 +1,14 @@
...
-    '156469054399404',   # Columbia Engineers Without Borders
-    '162282307296224',   # Columbia Financial Investment Group
-    '469888179740276',   # Columbia Financial Review
-    '249997198467774',   # Columbia Graduate Consulting Club
-    '682327028452194',   # Columbia International Relations Council and Association
-]
-
 
 def update_fb_events():
+    f = open('cal/fb_groups.yml')
```
Comment (natebrennand) [inline cal/fb.py:11]: please use a `with` statement here

File: cal/__init__.py [hunk 2]
```diff
@@ -66,13 +66,7 @@ def home():
...
 
-# TO REMOVE: Manual update route
-@app.route('/update')
-def update():
-    update_fb_events()
-    return jsonify({"success": True})
-
-
+# TO REMOVE
 @app.route('/events')
```
Comment (natebrennand) [inline cal/__init__.py]: this should have a decorator that depends on the `DEBUG` flag that either enables it or throws a 404

Comment (alanhdu) [inline cal/__init__.py]: We don't actually want to remove this route (although we probably will change our routing structure in the future). It's where React populates its initial event list.

File: cal/fb.py [hunk 3]
```diff
@@ -1,108 +1,14 @@
 from facebook import GraphAPI
 from cal.schema import db, User, Event
 from config import FACEBOOK_ACCESS_TOKEN
-import iso8601
+import iso8601, yaml
```
Comment (alanhdu) [inline cal/fb.py]: pep8 problem.

File: cal/fb.py (hunk 1 above)
Comment (alanhdu) [inline cal/fb.py:11]:
``` Python
with open("cal/fb_groups.yml") as f:
    page_ids = yaml.load(f).keys()
```

It works better w/ exceptions and things like that.

**Comment by alanhdu** [comment]
Couple of comments, and a rebase, and this should be good.

File: cal/fb.py (hunk 1 above)
Comment (davidhao3300) [inline cal/fb.py:11]: Could've sworn I changed it to that, good catch.

File: cal/__init__.py (hunk 2 above)
Comment (davidhao3300) [inline cal/__init__.py]: Got it

**Comment by alanhdu** [comment]
LGTM.

**Pull request merged**